tests/unit/spec_tests/test_group_spec.py

```diff
@@ -209,3 +209,29 @@ def test_update_attribute_spec(self):
         res = spec.get_attribute('attribute1')
         self.assertEqual(res.value, 5)
         self.assertEqual(res.dtype, 'int')
+
```

Comment:
Can you also test for the case when `Spec.name` is not set?

Reply:
Added.